platform/pc/service.c

```diff
@@ -446,11 +446,19 @@ void __attribute__((noreturn)) kernel_shutdown(int status)
 u64 total_processors = 1;
 
 #ifdef SMP_ENABLE
+/* hvm does not always properly initialize mxcsr register */
+static void init_mxcsr() {
+    u32 m = 0x1f80;
```

Comment:
I see that this is the reset value right out of the SDM - so maybe it's not necessary to break it down into a union of individual flags - but maybe a define for this constant with a reference to its origin would be in order?